lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable self-paced courses.
+    'ENABLE_SELF_PACED_COURSES': False,
```

Comment:
It seems like it would make the most sense to place that model in a new app in `common/djangoapps`, does that sound right to you?